Tensorflow lite build failed : ‘atomic’ in namespace ‘std’ does not name a type

This issue happened while building tensorflow lite into AWS Lambda docker

**System information**
- OS platform and distribution : [docker image lambci/lambda:build-python3.6](https://hub.docker.com/r/lambci/lambda/tags)
- TensorFlowLite pip package built from source
- Tensorflow version : master (currently on 496acff)
- Python version: 3.6
- Compiler version : 
  * clang version 3.6.2 (tags/RELEASE_362/final)
  * gcc version 4.8.5 20150623 (Red Hat 4.8.5-28) (GCC) 
  * ldd (GNU libc) 2.17

**Describe the problem**

Using this Dockerfile

```
FROM lambci/lambda:build-python3.6

WORKDIR /var/task

RUN yum update -y && yum -y install swig libjpeg-devel zlib-devel wget

RUN git clone https://code.googlesource.com/re2
WORKDIR /var/task/re2
RUN make && make test && make install && make testinstall

WORKDIR /var/task/

RUN wget https://github.com/google/googletest/archive/release-1.8.0.tar.gz &&\
    tar xf release-1.8.0.tar.gz
		
WORKDIR /var/task/googletest-release-1.8.0

RUN cmake -DBUILD_SHARED_LIBS=ON . &&\
    make &&\
		make install
		
WORKDIR /var/task
RUN pip install numpy
RUN git clone https://github.com/tensorflow/tensorflow
WORKDIR /var/task/tensorflow

RUN git checkout 496acff
RUN bash ./tensorflow/lite/tools/pip_package/build_pip_package.sh
```

It fails with : 

```
./tensorflow/lite/kernels/acceleration_test_util_internal.h: In function ‘absl::optional<T> tflite::GetAccelerationTestParam(std::string)’:
./tensorflow/lite/kernels/acceleration_test_util_internal.h:69:10: error: ‘atomic’ in namespace ‘std’ does not name a type
   static std::atomic<std::vector<ConfigurationEntry<T>>*> test_config_ptr;
```

--- a/tensorflow/lite/kernels/acceleration_test_util_internal.h
+++ b/tensorflow/lite/kernels/acceleration_test_util_internal.h
@@ -21,6 +21,7 @@ limitations under the License.
 #include <string>
 #include <utility>
 #include <vector>
+#include <atomic>
 
 #include "absl/types/optional.h"
 #include "re2/re2.h"

